app/models/course.rb

```diff
@@ -7,8 +7,12 @@ class Course < ApplicationRecord
   validates :level, presence: true
   validates :code, uniqueness: { scope: :provider_id }
 
-  scope :visible_to_candidates, -> { where(exposed_in_find: true) }
-  scope :applyable, -> { visible_to_candidates.where(open_on_apply: true) }
+  # Also filter on `exposed_in_find` as to hide courses that are hidden in Find,
+  # but are enabled by a support user in Apply
+  scope :open_on_apply, -> { where(open_on_apply: true, exposed_in_find: true) }
```

Comment:
It's mildly confusing that we have a scope and a flag called `open_on_apply`, and something is only *open* open if it's matched by the scope

Reply:
That's why we have the comment! Do you have another naming suggestion?

Reply:
Is it true to say that something which is only `open_on_apply` is actually open, because if it's not `applyable?` then `CourseChoicesController` will tell them they have to apply on UCAS.